gcp_variant_transforms/libs/variant_merge/move_to_calls_strategy.py

```diff
@@ -73,23 +106,12 @@ def get_merged_variants(self, variants):
                                  alternate_bases=variant.alternate_bases)
       merged_variant.names.extend(variant.names)
       merged_variant.filters.extend(variant.filters)
-      if (variant.quality is not None and
-          (merged_variant.quality is None or
-           merged_variant.quality < variant.quality)):
-        merged_variant.quality = variant.quality
-      additional_call_info = {}
-      if self._should_copy_filter_to_calls():
-        additional_call_info[ColumnKeyConstants.FILTER] = variant.filters
-      if self._should_copy_quality_to_calls():
-        additional_call_info[ColumnKeyConstants.QUALITY] = variant.quality
-      for info_key, info_value in variant.info.iteritems():
-        if self._should_move_info_key_to_calls(info_key):
-          additional_call_info[info_key] = info_value.data
-        else:
-          merged_variant.info[info_key] = info_value
-      for call in variant.calls:
-        call.info.update(additional_call_info)
-        merged_variant.calls.append(call)
+      merged_variant.quality = max(merged_variant.quality, variant.quality)
```

Comment:
nice! I didn't realize max works with None types :)

Reply:
Yeah it's pretty convenient! I think comparison operators work for most python built in types.